### PR TITLE
print confirmation tweaks

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](link-goes-here)

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by cantsin** [comment]
Opening up for review with the caveat that Aviva hasn't approved the PDF preview yet, so there might be further changes.

File: crt_portal/cts_forms/templates/forms/confirmation.html [hunk 1]
```diff
@@ -2,16 +2,22 @@
...
 {% block page_header %}
 <header class="crt-form-header">
   <div class="grid-container">
     <div class="grid-row grid-gap flex-child">
       <div class="tablet:grid-col-9 tablet:grid-offset-1">
+        <div class="doj-brand-print">
+          <img src="{% static "img/doj-logo-footer.png" %}" alt="" height="38" class="margin-right-105" />
+          <span>{% trans "United States Department of Justice" %}</span>
+          <span>{% trans "Civil Rights Division" %}</span>
+          <span class="doj-brand-url">www.civilrights.justice.gov</span>
```
Comment (Jkrzy) [inline crt_portal/cts_forms/templates/forms/confirmation.html:16]: Let's trim the `www` to keep this consistent with our social media images

```suggestion
          <span class="doj-brand-url">civilrights.justice.gov</span>
```

**Review by Jkrzy — APPROVED** [review]
:amaze: Thanks @cantsin!

**Review by timoballard — APPROVED** [review]
👍  tested in Edge/IE

**Review by Jkrzy — APPROVED** [review]
Thanks @cantsin and @timoballard!

**Pull request merged**